src/components/Settings/MenuButton/index.tsx

```diff
@@ -31,19 +33,20 @@ const Button = styled.button<{ isActive: boolean }>`
 `
 
 const IconContainer = styled(Row)`
-  padding: 6px 12px;
+  padding: 4px 4px 4px 12px;
   border-radius: 16px;
 `
 
 const IconContainerWithSlippage = styled(IconContainer)<{ displayWarning?: boolean }>`
   div {
-    color: ${({ theme, displayWarning }) => (displayWarning ? theme.deprecated_accentWarning : theme.neutral2)};
+    color: ${({ theme, displayWarning }) => (displayWarning ? theme.neutral2 : theme.neutral2)};
   }
 
-  background-color: ${({ theme, displayWarning }) =>
-    displayWarning ? theme.deprecated_accentWarningSoft : theme.surface2};
+  background-color: ${({ theme, displayWarning }) => (displayWarning ? theme.surface2 : theme.surface2)};
```

Comment:
This conditional has the same value on each side can  be removed altogether

Reply:
fixd